src/data/action-algorithms/data-source.js

```diff
@@ -0,0 +1,261 @@
+import {
+  ActionAlgorithm as coreActionAlgorithm,
+  Utils
+} from '@apollosproject/data-connector-rock'
+import {
+  get,
+  flattenDeep,
+} from 'lodash'
+import moment from 'moment-timezone'
+import ApollosConfig from '@apollosproject/config'
+
+const { ROCK_MAPPINGS, ROCK } = ApollosConfig
+const { createImageUrlFromGuid } = Utils
+
+export default class ActionAlgorithm extends coreActionAlgorithm {
+  expanded = true
+
+  /** Base Attrs and Methods from the Core DataSource */
+  baseAlgorithms = this.ACTION_ALGORITHIMS;
```

Comment:
```suggestion
  baseAlgorithms = this.ACTION_ALGORITHMS;
```

we spelled it correctly this time 😬